ggircs-app/app/pages/_app.tsx

```diff
@@ -1,108 +1,34 @@
-import React from "react";
-import { QueryRenderer } from "react-relay";
-import NextApp from "next/app";
-import { NextRouter } from "next/router";
-import { PageComponent } from "next-env";
-import { getRequest } from "relay-runtime";
-import { createEnvironment } from "lib/relay-environment";
-import ErrorBoundary from "lib/error-boundary";
-import LoadingSpinner from "components/LoadingSpinner";
-import ToasterHelper from "components/Layout/Toaster";
-import "react-toastify/dist/ReactToastify.min.css";
-import PageRedirectHandler from "components/Layout/PageRedirectHandler";
+import { AppProps } from "next/app";
+import { RelayEnvironmentProvider } from "react-relay/hooks";
+import { getInitialPreloadedQuery, getRelayProps } from "relay-nextjs/app";
+import { getClientEnvironment } from "../lib/relay/client";
+import "normalize.css";
+import SessionExpiryHandler from "components/Session/SessionExpiryHandler";
+import { Suspense } from "react";
+const clientEnv = getClientEnvironment();
+const initialPreloadedQuery = getInitialPreloadedQuery({
+  createClientEnvironment: () => getClientEnvironment()! as any,
```

Comment:
@matthieu-foucault I'm unclear what the type issue is here that required an `as any` to resolve. This is implemented the same way in cif with no type issues https://github.com/bcgov/cas-cif/blob/develop/app/pages/_app.tsx#L10. It looks to me like I've added the same types to the dev deps in the package.json as cif. Did I maybe miss something in a config file somewhere? (Same issue on line 27 below)

Reply:
`@types/react-relay` has a `"@types/relay-runtime" "*"` dependency which is causing some confusion. Depending on when `@types/react-relay` was installed in the first place, it will depend on a different version of `@types/relay-runtime`, and, depending on which version of `@types/relay-runtime` we require, there's a mismatch in the type definitions

Reply:
to resolve this. Delete your `"@types/relay-runtime@*"` entry from the `yarn.lock`, do a `yarn install` to that you end up with a single `"@types/relay-runtime@*", "@types/relay-runtime@^13.0.0"`.
Now when I did this it did not delete that old `@types/relay-runtime` dependency under `node_modules/@types/react-relay/node_modules`, so you might want to delete your `node_modules` as well for good measure.